lib/src/config_provider/spec_utils.dart

```diff
@@ -327,6 +328,97 @@ Map<String, ImportedType> makeImportTypeMapping(
   return typeMappings;
 }
 
+Type makePointerToType(Type type, int pointerCount) {
+  for (var i = 0; i < pointerCount; i++) {
+    type = PointerType(type);
+  }
+  return type;
+}
+
+String makePostfixFromRawVarArgType(List<String> rawVarArgType) {
+  return rawVarArgType
```

Comment:
Should we consider generating a C file with configurable imports instead? And run clang on it to get the mapping from C to Dart types.

For example: What if I use a struct as vararg that I did not transitively included otherwise?

Reply:
> What if I use a struct as vararg that I did not transitively included otherwise?

That will generate types with no reference, but the current approach allows also using types declaration in any external library as well (library-imports config). So i guess this is a  tradeoff.

Reply:
Hm, what about renames? If a user has made a rename from `MyStruct` in C to `PrefixedMyStruct` in Dart?

It feels slightly more robust to make it go through the whole pipeline, we would pick up all the features we have in ffigen. But maybe I am mistaken, what do you think?

Reply:
I feel it may be more difficult to trace back what renamed a struct than just looking at the generated bindings.

And while C allows structs/unions/typedefs to share the same name, these get renamed by ffigen due to name collision, although this is rare but still possible. The only way to then differentiate those is by having the user specify what sort of type it is with its original name in C.